.github/ISSUE_TEMPLATE/add-an-awesome-github-profile.yml

```diff
@@ -0,0 +1,14 @@
+name: "Add an Awesome GitHub Profile"
+description: "Add your awesome GitHub profile to the project"
+title: "[PROFILE ADDITION] <name here>"
+labels: []
```

Comment:
we could think of adding a profile add label here

Reply:
Yeah. That's a good idea. :)
Should we name it `profile addition`, `new profile`, or `new awesome profile`?

Reply:
I think profile addition is a good name